lighthouse-cli/scripts/run-smoke-tests.sh

```diff
@@ -3,34 +3,36 @@
 cd lighthouse-cli/test/fixtures && python -m SimpleHTTPServer 9999 &
 
 NODE=$([ $(node -v | grep -E "v4") ] && echo "node --harmony" || echo "node")
-offline200result="URL responds with a 200 when offline"
+config="$PWD/lighthouse-cli/test/fixtures/smoketest-config.json"
+flags="--audit-whitelist=works-offline,viewport --config-path=$config"
```

Comment:
I don't think we need both an audit-whitelist and a custom config path. The whitelist works to remove unnecessary audits from the default config, but if using a custom config we are already limiting the audits we want.